geode-core/src/main/java/org/apache/geode/distributed/Locator.java

```diff
@@ -253,7 +253,7 @@ private static Locator startLocator(int port, File logFile, InetAddress bindAddr
       java.util.Properties dsProperties, boolean peerLocator, boolean serverLocator,
```

Comment:
Amey, there is also a loadSharedConfig static variable in Locator that should be removed.